source/_integrations/sensor.markdown

```diff
@@ -22,7 +22,7 @@ The type of data a sensor returns impacts how it is displayed in the frontend. T
 - **None**: Generic sensor. This is the default and doesn't need to be set.
 - **apparent_power**: Apparent power in VA.
 - **aqi**: Air Quality Index (unitless).
-- **atmospheric_pressure**: Atmospheric pressure in cbar, bar, hPa, inHg, kPa, mbar, Pa or psi
+- **atmospheric_pressure**: Atmospheric pressure in cbar, bar, hPa, mmHg, inHg, kPa, mbar, Pa or psi
```

Comment:
This should also be added to our dev docs.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes

Reply:
thank you for the catch